src/ngraph/runtime/cpu/builder/matmul_bias.cpp

```diff
@@ -78,12 +78,12 @@ namespace ngraph
                             k,
                             1.0f,
                             static_cast<float*>(arg0_tensor),
-                            max(1UL, lda),
+                            max((size_t)1UL, lda),
```

Comment:
With the `size_t` there's no need for the `UL` so let's simplify to `1`.

Reply:
Don't do old-style C casts. The right way to do this is `max<size_t>(1, lda)`

Reply:
Yes. fixed it.